frontend/src/metabase-types/api/settings.ts

```diff
@@ -128,10 +128,11 @@ export interface TokenFeatures {
   whitelabel: boolean;
 }
 
-export interface SettingDefinition {
+export interface SettingDefinition<Value = unknown> {
```

Comment:
We won't be able to type `SettingDefinition` because its coming from the backend in a single list. Could we make `value: unknown` and remove the generic parameter?